lib/writeBinaryGltf.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 var fs = require('fs');
 var path = require('path');
+var async = require('async');
 var mkdirp = require('mkdirp');
 var getBinaryGltf = require('./getBinaryGltf');
+var writeSource = require('./writeSource');
 
 module.exports = writeBinaryGltf;
 
-function writeBinaryGltf(gltf, outputPath, createDirectory, callback) {
+function writeBinaryGltf(gltf, outputPath, embed, embedImage, createDirectory, callback) {

```

Comment:
This is starting to get unwieldy enough that I suggest changing this to take `options`.

See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#options-parameters